idea-plugin/src/main/resources/inspectionDescriptions/CoercionExplicitArgs.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This inspection help us making the coercions within the arguments of an expression to be explicit
```

Comment:
```suggestion
This inspection help us making the coercions within the arguments of an expression to be explicit.
```
I know we're not into final texts yet but let's try to keep syntax correct just in case some texts make it to the final version 👍 

Also: Isn't it possible to get these localized? It'd be weird to have English here and then Spanish in the rest of the UI.